.circleci/scripts/binary_populate_env.sh

```diff
@@ -78,8 +78,8 @@ TRITON_VERSION=$(cat $PYTORCH_ROOT/.ci/docker/triton_version.txt)
 TRITON_CONSTRAINT="platform_system == 'Linux' and platform_machine == 'x86_64' and python_version < '3.13'"
 if [[ "$PACKAGE_TYPE" =~ .*wheel.* &&  -n "${PYTORCH_EXTRA_INSTALL_REQUIREMENTS:-}" ]]; then
   # Only linux Python < 3.13 are supported wheels for triton
-  TRITON_REQUIREMENT="triton==${TRITON_VERSION}; ${TRITON_CONSTRAINT}"
-  if [[ -n "$PYTORCH_BUILD_VERSION" ]]; then
+  TRITON_REQUIREMENT="pytorch-triton==${TRITON_VERSION}; ${TRITON_CONSTRAINT}"
```

Comment:
This feels like something we shouldn't be doing?

Reply:
This job builds pytorch-triton package. Here is the workflow from 2.4:
https://github.com/pytorch/pytorch/actions/runs/10200015317/job/28218504485#step:8:51 and this is what we use in nightlies as well. For RC builds we use the package we build. Once we are happy with it, release to pypi happens this is when this particular line is reverted and we use triton package.

Reply:
Should we undo those changes on trunk than?

Reply:
Yes, agree. Will post a change with this to trunk. It will save us 1 cherry-pick during the release process.